src/waveform/guitick.cpp

```diff
@@ -28,6 +37,18 @@ void GuiTick::process() {
     m_cpuTime += elapsedS;
     m_pCOGuiTickTime->set(m_cpuTime);
 
+	if (m_lastUpdateTime2 + 3.00 < m_cpuTime)
+	{
+		tabletMode = (GetSystemMetrics(SM_CONVERTIBLESLATEMODE) == 0);
```

Comment:
Polling this seams to be not that effective. 
Isn't there a windows signal we can hook on? 

If not, we should remove this polling code to a separate class, connecting to the valueChanged signal of a 
"[Master]", "guiTick50ms" ControlObjectSlave 
This code should be mad conditional so that it is only included on Win builds.